spec/controllers/api/v2/quotas_controller_spec.rb

```diff
@@ -164,7 +155,7 @@
           year: [
             Time.zone.today.year.to_s,
           ],
-          include: 'quota_balance_events,measures,measures.geographical_area',
+          include: 'quota_balance_events',
```

Comment:
We're missing an example covering the additional includes case?

Reply:
the other case:
`'when not specifying an includes list in the query params'`
is already in place (and working).